sockeye/inference.py

```diff
@@ -392,6 +393,7 @@ def load_models(context: mx.context.Context,
                                    restrict lexicon).
     :param forced_max_output_len: An optional overwrite of the maximum output length.
     :param override_dtype: Overrides dtype of encoder and decoder defined at training time to a different one.
+    :param output_scores: Whether the scores will be needed as outputs.
```

Comment:
can we make the parameter and its description more descriptive? I.e. what do we mean by scores exactly? Normalized negative log probabilities.